docs/UserGuide.md

```diff
@@ -28,178 +28,228 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
-**Notes about the command format:**<br>
+Here are some **key points** to keep in mind when using commands:
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+- You should supply all the parameters (words in `UPPER_CASE`) that are needed for a command.<br>
+  e.g. in `add person n/NAME`, `NAME` is a parameter which can be used as `add person n/John Doe`.
 
-* Items in square brackets are optional.<br>
+- Items in square brackets are optional.<br>
   e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
 
-* Items with `…`​ after them can be used multiple times including zero times.<br>
+- Items with `…`​ after them can be used multiple times including zero times.<br>
   e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
 
-* Parameters can be in any order.<br>
+- You can add parameters in any order.<br>
   e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 
-* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
+- If you add extraneous parameters for commands that do not take in parameters (such as `help`, `list person`, `list appt`, `exit` and `clear`), they will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
-* If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
-</box>
+- If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
 
-### Viewing help : `help`
+---
 
-Shows a message explaning how to access the help page.
+## Command Overview
 
-![help message](images/helpMessage.png)
+The following sections describe the various commands available in the DocTrack platform:
 
-Format: `help`
+- [Person Commands](#person-commands)
+- [Appointment Commands](#appointment-commands)
+- [General Commands](#general-commands)
+
+---
 
+### Person Commands
 
-### Adding a person: `add`
+A **person** is a patient with several data fields: a name, a phone number, an email, an address and optional tags. These patients can be uniquely identified by their **patient ID (PID)** for easy reference. DocTrack allows you interact with patient information through different commands, which can be seen below.
```

Comment:
Perhaps can keep it consistent for the above as well. Choose whether to use "fields" or "parameters". I think fields would be a more user-friendly language? Since "parameters" has some math connotation

Reply:
Related to first comment